packages/office-ui-fabric-react/src/components/ResizeGroup/examples/ResizeGroup.VerticalOverflowSet.Example.tsx

```diff
@@ -16,35 +16,28 @@ export interface IOverflowData {
   cacheKey?: string;
 }
 
-function generateData(count: number, cachingEnabled: boolean, checked: boolean): IOverflowData {
+const generateData = (count: number, cachingEnabled: boolean, checked: boolean): IOverflowData => {
   const icons = ['Add', 'Share', 'Upload'];
-  const dataItems = [];
   let cacheKey = '';
-  for (let index = 0; index < count; index++) {
-    const item = {
+  const dataItems = Array.from({ length: count }).map((item, index) => {
```

Comment:
`Array.from` doesn't work in IE 11, so you need to go back to the previous approach.

Reply:
suggested to use `createArray` which is safer and a bit more simpler anyway; he'll find/replace other references too because he mentioned he had added that in some other places.

Reply:
Huh I forgot that existed